backend/src/vrr/service/api.service.ts

```diff
@@ -1,10 +1,10 @@
 /* eslint-disable @typescript-eslint/naming-convention */
 import { Injectable } from "@nestjs/common";
 import { VRR_TEST_API_BASE_URL } from "@oeffis/vrr_client";
+import { LocationType as VrrLocationType } from "@oeffis/vrr_client/dist/vendor/VrrApiTypes";
 import { VrrClientBase } from "@oeffis/vrr_client/dist/VrrClientBase";
-import { RealtimeTripStatus, LocationType as VrrLocationType } from "@oeffis/vrr_client/dist/vendor/VrrApiTypes";
-import { LegRealtimeTripStatus } from "vrr/entity/legRealtimeTripStatus.entity";
 import { LocationType } from "vrr/entity/locationType.entity";
+import { LegInfoPriority } from "../entity/legInfoPriority.entity";
 
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
While you're at it: Can you remove this line? It has no effect. There is no any in the next line.